astro/src/content/docs/release-notes/index.mdx

```diff
@@ -45,9 +46,24 @@ import { YouTube } from '@astro-community/astro-embed-youtube';
 
 Looking for release notes older than 1.44.0? Look in the [release notes archive](/docs/release-notes/archive). Looking to be [notified of new releases?](/docs/operate/roadmap/releases#release-notifications) <span class="not-prose no-underline"><a class="ml-2" href="/docs/releases.xml"><i class="fas fa-xs fa-rss text-orange-700 text-2xl" width="50px" /></a></span>
 
-<ReleaseNoteHeading version='1.58.0' releaseDate='July 21, 2025' />
-### Known Issues
-* A bug exists that prevents the Groups page from loading properly. If you use groups, it is recommended that you not upgrade to version `1.58.0` and wait instead for the `1.58.1` patch release.
+<ReleaseNoteHeading version='1.58.2' releaseDate='July 23, 2025' name='Delegation Dolphin' />
```

Comment:
Only `.0` releases are named, this is just a patch, so the name should be removed.